source/testing/acceptance.md

```diff
@@ -130,13 +130,11 @@ previous steps in the test have completed.
 
 Synchronous helpers are performed immediately when triggered.
 
-* [`currentPath()`][6]
```

Comment:
Not sure if I missed it, but it looks like `currentPath` has been removed:
https://github.com/emberjs/ember-test-helpers/blob/37c4139ee73c19a46670268825881d81eb37d663/tests/unit/setup-application-context-test.js#L6-L14